src/compass/utils/validate_cslc.py

```diff
@@ -0,0 +1,151 @@
+import argparse
+import json
+import os
+
+import numpy as np
+from osgeo import gdal
+
+
+def cmd_line_parser():
+    """
+    Command line parser
+    """
+
+    parser = argparse.ArgumentParser(description="""
+                                     Validate reference and generated (secondary) S1 CSLC products""",
```

Comment:
```suggestion
    parser = argparse.ArgumentParser(description="Validate reference and generated (secondary) S1 CSLC products",
```

To remove the long whitespaces.